packages/cli/src/commands/init/init.ts

```diff
@@ -170,6 +185,30 @@ async function createFromTemplate({
         loader,
         root: projectDirectory,
       });
+
+      if (process.platform === 'darwin') {
+        const installPodsValue = String(installCocoaPods);
```

Comment:
basically you can have a multiple values here:
`true` - when you use `--install-pods` 
`"true"`/`"false"` - when `--install-pods true`/ `--install-pods false`
`undefined` - when flag is not used

converting to a string seems like the fastest way to avoid building complicated if statements

Reply:
Let's change the type to `string | boolean` then